### ISSUE TITLE
Corporate proxy settings for crawler

### ISSUE THREAD

**Issue by azat-alimov-db** [change-metrics/monocle#1090]
Hello,

Is there a way to configure crawler to collect metrics from Github, via corporate proxy?

**Comment by morucci** [comment]
No actually this is not supported.

I guess we could enable the proxy via the env vars (https://hackage.haskell.org/package/http-client-0.7.15/docs/Network-HTTP-Client.html#g:5) ?

**Comment by TristanCacqueray** [comment]
@azat-alimov-db could you try setting the `https_proxy` environment variable for the cralwer process?

**Comment by morucci** [comment]
@TristanCacqueray yes you are right. It seems to be supported already by default (I did a quick try with Squid). The crawler reads `https_proxy=<hostname>:3128` and forward requests to the proxy.

**Comment by azat-alimov-db** [comment]
Thank you that works.
